### PR TITLE
Established connection with postgresql database, created and mapped e…

### PR DESCRIPTION
**Pull request by psamula**
backend skeleton

### CODE REVIEW

**Comment by Bartosz-Slowik** [comment]
Its good because its initial commit but please try to make commits smaller than 100 files xd

File: src/backend/main/java/today/meetnow/config/FlywayConfig.java
Comment (Bartosz-Slowik) [inline src/backend/main/java/today/meetnow/config/FlywayConfig.java:1]: why is this commented?

File: src/backend/main/resources/db/migration/V1__initTables_PS.sql
Comment (Bartosz-Slowik) [inline src/backend/main/resources/db/migration/V1__initTables_PS.sql:1]: This won't duplicate user@user each time?

File: src/backend/main/resources/db/migration/V4__create_event_related_tables.sql
Comment (Bartosz-Slowik) [inline src/backend/main/resources/db/migration/V4__create_event_related_tables.sql:1]: Are we concidering multiple hosts of the same event?

File: target/classes/db/migration/V1__initTables_PS.sql
Comment (Bartosz-Slowik) [inline target/classes/db/migration/V1__initTables_PS.sql:1]: why is this file duplicated in target? Should we even have target on git?

**Comment by psamula** [comment]
> Its good because its initial commit but please try to make commits smaller than 100 files xd

It's because I forgot to add some dirs and files to .gitignore (fixed now), the commit itself is not that big

File: target/classes/db/migration/V1__initTables_PS.sql
Comment (psamula) [reply target/classes/db/migration/V1__initTables_PS.sql:1]: fixed (gitignore, git rm -r -cached)

File: src/backend/main/resources/db/migration/V4__create_event_related_tables.sql
Comment (psamula) [reply src/backend/main/resources/db/migration/V4__create_event_related_tables.sql:1]: > Are we concidering multiple hosts of the same event?
nope, one host one event

File: src/backend/main/resources/db/migration/V1__initTables_PS.sql
Comment (psamula) [reply src/backend/main/resources/db/migration/V1__initTables_PS.sql:1]: fixed (commit no 5)

File: src/backend/main/java/today/meetnow/config/FlywayConfig.java
Comment (psamula) [reply src/backend/main/java/today/meetnow/config/FlywayConfig.java:1]: gotten rid of

File: src/backend/main/resources/db/migration/V1__initTables_PS.sql
Comment (Bartosz-Slowik) [reply src/backend/main/resources/db/migration/V1__initTables_PS.sql:1]: Oh its validated.

File: src/backend/main/resources/db/migration/V4__create_event_related_tables.sql
Comment (Bartosz-Slowik) [reply src/backend/main/resources/db/migration/V4__create_event_related_tables.sql:1]: Then whats the purpose of this table?

File: src/backend/main/resources/db/migration/V4__create_event_related_tables.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+CREATE TABLE event (
+                       id SERIAL PRIMARY KEY,
+                       title VARCHAR(255),
+                       description TEXT,
+                       type VARCHAR(255),
+                       image VARCHAR(255),
+                       coordinates POINT
+);
+CREATE TABLE host (
```
Comment (Bartosz-Slowik) [inline src/backend/main/resources/db/migration/V4__create_event_related_tables.sql:9]: Then whats the purpose of this table?

File: src/backend/main/resources/db/migration/V4__create_event_related_tables.sql
Comment (psamula) [reply src/backend/main/resources/db/migration/V4__create_event_related_tables.sql:1]: As a concept of a user who has a behavior and state characteristic only for an event host. Right now the table and entity seem redundant but it's all to be added in the future I guess? Right now it's only a skeleton and can be extended without the need for modification.
Same for the participant table.

**Comment by Bartosz-Slowik** [comment]
Lgtm , imo we should merge it so other people can start working from a backend skeleton

**Pull request merged**